pkg/operation/cloudbotanist/azurebotanist/cloud_config.go

```diff
@@ -1,26 +0,0 @@
-// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package azurebotanist
-
-import (
-	"github.com/gardener/gardener/pkg/operation/common"
-)
-
-// GenerateCloudConfigUserDataConfig generates values which are required to render the chart shoot-cloud-config properly.
-func (b *AzureBotanist) GenerateCloudConfigUserDataConfig() *common.CloudConfigUserDataConfig {
-	return &common.CloudConfigUserDataConfig{
-		ProvisionCloudProviderConfig: true,
```

Comment:
I don't see this implemented in https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-azure/pkg/webhook/controlplane/ensurer.go#L123

/cc @mkoynov

Reply:
We will add it, thanks for noticing!